cpp/ps0001/p0056.so1.cpp

```diff
@@ -4,13 +4,14 @@ using namespace std;
 
 int main()
 {
-
-    int x , y , o ;
-
+    // prompting user for 3 numbers to add 1/2
+    unsigned short num1, num2, num3;
     cout << "please enter x and y and o \n";
-    cin >> x >> y >> o ;
+    cin >> num1 >> num2 >> num3;
 
-    cout << "total = " << x + y + o << "\n" ;
+    // outputting sum of the 3 numbers 3/3
+    cout << "total = " << num1 + num2 + num3 << "\n" ;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
```